lm_eval/evaluator.py

```diff
@@ -38,6 +41,9 @@ def simple_evaluate(
     write_out: bool = False,
     log_samples: bool = True,
     gen_kwargs: str = None,
+    weight_by_size: bool = False,
+    task_manager: TaskManager = None,
+    verbosity: str = "INFO",
```

Comment:
If we are passing `verbosity` into simple_evaluate, we should respect it for our logger too and set verbosity within `simple_evaluate()`